README.md

```diff
@@ -56,6 +54,44 @@ brew install foundry
 
 Run `foundry version` to verify it's installed correctly.
 
+## Getting Started
+
+Clone this sample to your local system, or [download as a zip file](https://github.com/CrowdStrike/foundry-sample-scalable-rtr/archive/refs/heads/main.zip).
+
+```shell
+git clone https://github.com/mraible/foundry-sample-scalable-rtr
+cd foundry-sample-scalable-rtr
+```
+
+Log in to your Foundry instance:
```

Comment:
I'm not certain, but I believe some sample apps don't require all permissions. I agree it might be less friction to tell everyone to select all the permissions, but I've noticed that sometimes the option is disabled. If we tell people to select an option that is unavailable to them, it might cause even more confusion. 

The reason I added this in here is because I experienced issues when installing this sample. In fact, I still haven't succeeded in deploying it and adding data to it. I plan to try again in the near future.

Reply:
Yah I suspect you're right that the various sample apps require different permissions. And I wasn't suggesting we tell users to add all the available permissions for all the sample apps - just that they can use an existing CLI profile and add the needed permissions to it if required. But maybe I'm making a faulty assumption here, that app devs would use only a single CLI profile per CID.